src/overlaybd/zfile/format_spec.md

```diff
@@ -22,12 +22,13 @@ The format of header is described as below. All fields are little-endian.
 | magic0  |       0        |      8       | "ZFile\0\1" (and an implicit '\0') |
 | magic1  |       8        |      16      | 74 75 6A 69, 2E 79 79 66, 40 41 6C 69, 62 61 62 61 |
 |  size   |      24        |   uint32_t   | size of the header struct (108), excluding the tail padding |
-| reserved|      28        |      4       | reserved space, should be 0 |
+| digest |      28        |      uint32_t       | checksum for the range 32-511 bytes in header |
 | flags   |      32        |   uint64_t   | bits for flags* (see later for details) |
 | index_offset | 40        |   uint64_t   | index offset |
-| index_size   | 48        |   uint64_t   | size of the index section, possibly compressed|
+| index_size   | 48        |   uint64_t   | size of the index section, possibly compressed base on flags |
 | original_file_size | 56  |   uint64_t   | size of the orignal file before compression |
-| reserved|      64        |      8       | reserved space, should be 0 |
```

Comment:
is it OK to (re)use these reserved space?

Reply:
the reserved space can be reused if set_digest_enable() in flags, or ignore it.